frontend/src/app/accountCreation/MfaTotp/MfaTotp.tsx

```diff
@@ -26,15 +54,22 @@ export const MfaTotp = (props: Props) => {
           app.
         </p>
         <div className="display-flex flex-column flex-align-center">
-          {props.qrCode ? (
+          {qrCode !== "" ? (
```

Comment:
Isn't `qrCode` guaranteed to exist at this point because we returned early in line 29? https://github.com/CDCgov/prime-simplereport/pull/1817/files#diff-212d61abf5384f41aff05b423eae28c1dfb1d97edece165deb1322078c5ab20bR29

Reply:
Yeah good catch! I wrote this before I implemented the `LoadingCard` component. Will fix

Reply:
Fixed in c7c755e